data/json/mapgen/dispensary.json

```diff
@@ -448,5 +449,66 @@
         { "item": "cannabis", "x": 20, "y": 15 }
       ]
     }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "dispensary_roof_2" ],
+    "weight": 100,
+    "object": {
+      "fill_ter": "t_tile_flat_roof",
+      "rows": [
+        "                        ",
+        "                        ",
+        "  44444444444444444444  ",
+        "  |.:................3  ",
+        "  |..................3  ",
+        "  |..................3  ",
+        "  |..................3  ",
+        "  |..................3  ",
+        "  |..................3  ",
+        "  |.....x............3  ",
+        "  |..................3  ",
+        "  |..................3  ",
+        "  |..................3  ",
+        "  |..................3  ",
+        "  5..................3  ",
+        "  |.....x............3  ",
+        "  |................&.3  ",
+        "  |..................3  ",
+        "  |2222222222...222223  ",
+        "             #>###      ",
+        "             ###        ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "terrain": {
+        ".": "t_tile_flat_roof",
+        " ": "t_open_air",
+        "|": "t_gutter_west",
+        "2": "t_gutter_south",
+        "3": "t_gutter_east",
+        "4": "t_gutter_north",
+        "5": "t_gutter_drop",
+        "#": "t_grate",
+        ">": "t_ladder_down"
+      },
+      "furniture": { "&": "f_roof_turbine_vent", ":": "f_vent_pipe" },
+      "place_items": [ { "item": "roof_trash", "x": [ 4, 15 ], "y": [ 5, 15 ], "chance": 50, "repeat": [ 1, 3 ] } ],
+      "place_nested": [
+        {
+          "chunks": [ [ "roof_2x2_golf", 20 ], [ "roof_2x2_utilities_b", 50 ], [ "null", 50 ], [ "roof_3x3_wine", 20 ] ],
```

Comment:
Nested mapgen `roof_3x3_wine` doesn't exist anywhere else in the files.

Reply:
Sorry, got ahead of myself.  This is dependent on Adds antique store roof #28859 for the roof_3x3_wine mapgen.  I've edited my description.  Let me know if I need to link them more explicitly.